main.go

```diff
@@ -334,7 +338,7 @@ func main() {
 					IncludeOptionalMetadata: withOptionalMetadataFlag,
 					IncludeStatus:           withStatusFlag,
 				}
-				resp, err := client.GetInstitutionByIDWithOptions(instID, opts)
+				resp, err := client.GetInstitutionByIDWithOptions(instID, []string {"US"}, opts)
```

Comment:
New `plaid-go` API required this additional argument - I figure just hard-coding `"US"` is fine?